docs/serverless/assistant/connect-to-vertex.mdx

```diff
@@ -0,0 +1,68 @@
+---
+slug: /serverless/security/connect-to-google-vertex
+title: Connect to Google Vertex
+description: Set up a Google Vertex LLM connector.
+tags: ["security", "overview", "get-started"]
+status: in review
+---
+
+# Connect to Google Vertex AI
+
+This page provides step-by-step instructions for setting up a Google Vertex AI connector for the first time. This connector type enables you to leverage Vertex AI's large language models (LLMs) within ((kib)). You'll first need to enable Vertex AI, then generate an API key, and finally configure the connector in ((kib)).
+
+<DocCallOut title="Important"> 
+Before continuing, you should have an active project in one of Google Vertex AI's [supported regions](https://cloud.google.com/vertex-ai/docs/general/locations#feature-availability). 
+</DocCallOut>
+
+## Emable the Vertex AI API
+
+1. Log in to the GCP console and navigate to **Vertex AI → Vertex AI Studio → Overview**. 
+2. If you're new to Vertex AI, the **Get started with Vertex AI Studio** popup appears. Click **Vertex AI API**, then click **ENABLE**.
+
+The following video demonstrates these steps.
+
+<DocVideo source="vidyard" id="vFhtbiCZiKhvdZGy2FjyeT"/>
```

Comment:
Not sure why, but all videos on the page render as screenshots.

Reply:
They render correctly if you refresh, in my experience 🤷